crates/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -180,26 +180,10 @@ impl Environment for ManagedEnvironment {
     /// WIP!
     /// TODO: errors!
     fn manifest_content(&self) -> Result<String, EnvironmentError2> {
-        let metadata_content = self
-            .floxmeta
-            .git
-            .show(&format!(
-                "{}:env.json",
-                branch_name(&self.system, &self.pointer, &self.path)?
-            ))
-            .unwrap();
-        let metadata: Value = serde_json::from_slice(metadata_content.as_bytes()).unwrap();
-        let current_gen = metadata["currentGen"].as_str().unwrap();
-        let manifest = self
-            .floxmeta
-            .git
-            .show(&format!(
-                "{}:{}/env/manifest.toml",
-                branch_name(&self.system, &self.pointer, &self.path)?,
-                current_gen
-            ))
-            .unwrap();
-        Ok(manifest.to_string_lossy().into())
+        let generations = self.generations();
+
+        let manifest = generations.current_gen_manifest().unwrap();
```

Comment:
nonblocking: is all that's left to get rid of the
```
    /// WIP!
    /// TODO: errors!
```
removing this unwrap? Seems like that might be worth it